graph/pkg/service/v0/groups.go

```diff
@@ -54,3 +139,162 @@ func (g Graph) GetGroup(w http.ResponseWriter, r *http.Request) {
 	render.Status(r, http.StatusOK)
 	render.JSON(w, r, group)
 }
+
+// DeleteGroup implements the Service interface.
+func (g Graph) DeleteGroup(w http.ResponseWriter, r *http.Request) {
+	groupID := chi.URLParam(r, "groupID")
+	groupID, err := url.PathUnescape(groupID)
+	if err != nil {
+		errorcode.InvalidRequest.Render(w, r, http.StatusBadRequest, "unescaping group id failed")
```

Comment:
Is a `return` missing here?

Reply:
yeah, thanks. @kobergj  Already notices those